packages/theming/atlas/src/themesource/atlas_core/web/core/base/_spacing.scss

```diff
@@ -9,441 +9,539 @@
+            $is_important: #{$important-spacing-value}
+        );
+    }
+    .spacing-inner-horizontal-layout {
+        @include layout-spacing(
+            $type: padding,
+            $direction: left,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+        @include layout-spacing(
+            $type: padding,
+            $direction: right,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: all,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-top-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: top,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-right-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: right,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-bottom-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: bottom,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-left-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: left,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    .spacing-outer-vertical-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: top,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+        @include layout-spacing(
+            $type: margin,
+            $direction: bottom,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+    .spacing-outer-horizontal-layout {
+        @include layout-spacing(
+            $type: margin,
+            $direction: left,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+        @include layout-spacing(
+            $type: margin,
+            $direction: right,
+            $device: responsive,
+            $is_important: #{$important-spacing-value}
+        );
+    }
+
+    // Scroll container spacing
+    .mx-scrollcontainer .mx-placeholder {
+        width: 100%;
+        height: 100%;
+
+        .mx-layoutgrid,
+        .mx-layoutgrid-fluid {
+            @include layout-spacing($type: padding, $direction: all, $device: responsive);
+
+            .mx-layoutgrid,
+            .mx-layoutgrid-fluid {
+                padding: 0;
+            }
+        }
+    }
```

Comment:
Maybe I missed it in the large diff, but where did this come from? 🤔

Reply:
from `_reset.scss`, talked with @cjfhodges about and agreed to move it